docs/ceph-csi-upgrade.md

```diff
@@ -2,7 +2,8 @@
 
 - [Ceph-csi Upgrade](#ceph-csi-upgrade)
   - [Pre-upgrade considerations](#pre-upgrade-considerations)
-  - [Upgrading from v1.2.x to v2.0.0](#upgrading-from-v12x-to-v200)
+  - [Upgrading from v1.2.x to v2.0.1](#upgrading-from-v12x-to-v201)
```

Comment:
maybe drop the last component of the version? Use v2.0.x as minor 2.0 updates should not need any different steps